src/browser/provider/built-in/dedicated/base.js

```diff
@@ -8,6 +8,16 @@ export default {
 
     isMultiBrowser: false,
 
+    supportMultipleWindows: true,
```

Comment:
If we need some special actions to support this functionality, it should be `false` by default, shouldn't it?

Reply:
It's `true` only for the dedicated browser providers: `Chrome`, `Firefox` and `Edge-Chromium`, because the src/browser/provider/built-in/dedicated/base.js file is common only for the dedicated browser providers (not for all).
By default, it's `false` for other browsers.

Reply:
My shame, that's right